src/main/java/org/apache/commons/text/CaseUtils.java

```diff
@@ -91,10 +91,7 @@ public static String toCamelCase(String str, final boolean capitalizeFirstLetter
             final int codePoint = str.codePointAt(index);
 
             if (delimiterSet.contains(codePoint)) {
-                capitalizeNext = true;
-                if (outOffset == 0) {
-                    capitalizeNext = false;
-                }
+                capitalizeNext = (outOffset != 0);
```

Comment:
No need for parentheses.

Reply:
@garydgregory OK, parentheses eliminated.